keps/sig-scheduling/1923-prefer-nominated-node/README.md

```diff
@@ -120,8 +120,8 @@ Following tests will be covered or considered:
   - preempt the victim pods on the nominated node
   - check pod will be scheduled on the nominated node
 - **Benchmark Tests**: A benchmark test which compares the performance before and after the change.
-  The performance improvement is visible by benchmark of `scheduling_algorithm_predicate_evaluation_seconds`.
-  Other benchmark will be created on-demand along with the code review process.
+  The performance improvement is visible by benchmark of `scheduler_framework_extension_point_duration_seconds{extension_point="Filter"}` in a large cluster
+  where preemption is expected to happen frequently.
```

Comment:
graduation criteria for beta are missing

Reply:
Done

Reply:
@chendave you need to add graduation criteria for Beta in the section below under `### Graduation Criteria`.

Reply:
thanks @ahg-g , just added.